tools/common/Driver.cs

```diff
@@ -348,5 +349,34 @@ internal static string GetFullPath ()
 			}
 		}
 
+		static void SetCurrentLanguage ()
+		{
+			// There's no way to change the current culture from the command-line
+			// without changing the system settings, so honor LANG if set.
+			// This eases testing mtouch/mmp with different locales significantly,
+			// and won't run into issues where changing the system language leaves
+			// the tester with an incomprehensible system.
+			var lang_variable = Environment.GetEnvironmentVariable ("LANG");
+			if (string.IsNullOrEmpty (lang_variable))
+				return;
+
+			// Mimic how mono transforms LANG into a culture name:
+			// https://github.com/mono/mono/blob/fc6e8a27fc55319141ceb29fbb7b5c63a9030b5e/mono/metadata/locales.c#L568-L576
+			var lang = lang_variable;
+			var idx = lang.IndexOf ('.');
+			if (idx >= 0)
+				lang = lang.Substring (0, idx);
+			idx = lang.IndexOf ('@');
+			if (idx >= 0)
+				lang = lang.Substring (0, idx);
+			lang = lang.Replace ('_', '-');
+			try {
+				var culture = CultureInfo.GetCultureInfo (lang);
+				if (culture != null)
+					CultureInfo.DefaultThreadCurrentCulture = culture;
```

Comment:
I'm a bit worried that this could _provoke_ other issues (or differences) on some computers.
Now that's the right thing to do (setting the culture) but I think we should print this out in the logs whenever we set `CultureInfo.DefaultThreadCurrentCulture` to something.